README.md

```diff
@@ -101,9 +101,12 @@ zoxide init fish | source
 
 ### Environment variables
 
-- `$_ZO_DATA`: sets the location of the database (default: `~/.zo`)
+- `$_ZO_DATA_DIR`: directory where `zoxide` will store its data files (default:
+    platform-specific; see the [`dirs` documentation] for more information)
```

Comment:
I think it would be better to use this link (`https://docs.rs/dirs/latest/dirs/fn.data_local_dir.html`) instead, so that it stays up-to-date.

Reply:
For brevity, I think it would be better to change this line to "see here for more" and include an inline link.

Reply:
I'll change it to point to latest.

By "see here for more" and inline link, do you mean: `see [here](insert url here) for more`? Or do you mean: `see here for more: <insert url here>`?

The current situation is already an inline link; rendered: https://github.com/ajeetdsouza/zoxide/blob/52ae1b03bcdcb148904936bc6d6f5c0ebd334372/README.md#environment-variables.

Reply:
I think `see [here](insert url here) for more` would be better. By inline link, I meant that if the text is changed to `here`, it wouldn't make much sense to separate the link out. Should have been more clear, my bad!

Reply:
All good. But I don't agree that `see [here](url) for more` is better than `see the [dirs doumentation] for more`, even in the interest of brevity. We don't have nearly enough doc comments to excuse short documentation in the readme, IMHO.

`see [here]` just sends you blindly to another page (ignore the fact that hovering shows URLs is most modern browsers); "where is 'here'? Why won't they tell me what this link is? Am I going  to be sent to a phishing page? Do I trust this project?" Hyperbolic, but hopefully it gets my point across.

`see the [dirs documentation]`, on the other hand, tells you exactly where you're going: to read the `dirs` documentation.

Reply:
That's a fair argument!